### PR TITLE
OPS-0: Added default repo 

### PR DESCRIPTION
**Pull request by vikkasyousaf**

### CODE REVIEW

File: library/parse_helm_repositories.py [hunk 1]
```diff
@@ -15,7 +15,6 @@ def parse_helm_repositories_output(text):
     reg = re.compile(r"^(?P<name>[^\s\t]+)[\s\t]+(?P<url>[^\s\n]+)$")
 
     # Loop through all the lines with the following format:
-    # stable             	https://kubernetes-charts.storage.googleapis.com
```
Comment (czerasz-flaconi) [inline library/parse_helm_repositories.py]: No need to remove this. This is just an example to understand the format better

File: library/test_parse_helm_repositories.py [hunk 2]
```diff
@@ -9,18 +9,13 @@ class parse(unittest.TestCase):
     def test_parse_helm_repositories_output(self):
         helm_repositories_output = """
 NAME               	URL
-stable             	https://kubernetes-charts.storage.googleapis.com
```
Comment (czerasz-flaconi) [inline library/test_parse_helm_repositories.py]: Why are You removing this from the test?

**Review by czerasz-flaconi — CHANGES_REQUESTED** [review]

File: library/test_parse_helm_repositories.py (hunk 2 above)
Comment (vikkasyousaf) [reply library/test_parse_helm_repositories.py]: Reverted

**Review by czerasz-flaconi — APPROVED** [review]
Well done! Good work! :+1:

**Pull request merged**